tests/gen_testfiles_scripts/mupairproduction.py

```diff
@@ -4,8 +4,6 @@
 
 particle_defs = [
     pp.particle.MuMinusDef(),
```

Comment:
The parametrization of the production of a muon pair by a charged lepton as the primary particle is only implemented for muons as primary particles, not for electrons and taus; the reason it was deleted from this test.
Therefore I created issue #257 targeting this.